src/components/Timer/Timer.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+
+export type TimerProps = {
+  className?: string;
+  unixTimestamp: number;
+  timerDisabled: boolean;
+  onTimerComplete: () => void;
+}
+
+export const Timer = ({
+  unixTimestamp = (new Date()).getTime() + 30000,
+  onTimerComplete,
+  timerDisabled,
+}: TimerProps) => {
+  const [minutes, setMinutes] = useState<number>(0);
+  const [seconds, setSeconds] =  useState<number>(0);
+
+  
+  useEffect(() => {
+    const now = new Date();
+    const unixDate: Date = new Date(unixTimestamp * 1000);
+    const distance = (unixDate.getTime() - now.getTime()) / 1000;
```

Comment:
why do you have `unixTimestamp * 1000` on line 21, shouldn't the unixTimestamp already be in milliseconds? I'm not exactly sure if the expiry we get back in our orders is in seconds/ms. Regardless, if the component doesn't receive a unixTimestamp param, the timestamp is in ms and the useEffect is still multiplying by 1000.

Reply:
Yep, unixTimestamp is in s, not ms. I'll reflect this change in the next commit :)